File: skiCentar/skiCentar.Model/LiftLocation.cs
namespace skiCentar.Model
{
    public class LiftLocation
    {
        public int Id { get; set; }

        public int? LiftId { get; set; }

        public decimal? LocationX { get; set; }

        public decimal? LocationY { get; set; }

        //public virtual Lift? Lift { get; set; }
    }
}

File: skiCentar/UI/skicentar_desktop/lib/models/lift-location.dart

import 'package:json_annotation/json_annotation.dart';
import 'package:skicentar_desktop/models/location.dart';

part 'lift-location.g.dart';
@JsonSerializable()
class LiftLocation implements Location{
  int? id;
  double? locationX;
  double? locationY;

  LiftLocation({this.id,this.locationX,this.locationY});

  factory LiftLocation.fromJson(Map<String,dynamic> json) => _$LiftLocationFromJson(json);

  Map<String,dynamic> toJson() => _$LiftLocationToJson(this);
}